File: app/src/main/java/com/tda/app/data/model/response/UserResponse.java
package com.tda.app.data.model.response;


import java.sql.Timestamp;
public class UserResponse {
    public long userId;
    public String firstname;
    public String lastname;
    public String email;
    public String phone;
    public String avatar;
    public boolean status;
    public Timestamp createDate;
    public Timestamp updateDate;
}

File: app/src/main/java/com/tda/app/model/response/UserResponse.kt
package com.tda.app.model.response

import java.sql.Timestamp

class UserResponse(
    val userId: Long,
    val firstname: String,
    val lastname: String,
    val email: String,
    val phone: String,
    val avatar: String,
    val status: Boolean,
    val createDate: Timestamp,
    val updateDate: Timestamp,
)